include/RED4ext/Scripting/Natives/CResource.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <cstdint>
```

Comment:
To satisfy the formatter I would say to add a new space after this include.

Reply:
I think it wants all std includes after RED4ext?
How it's in the project:
```
#include <cstdint>

#include <RED4ext/Common.hpp>
```
What formatter suggests:
```
#include <RED4ext/Common.hpp>

#include <cstdint>
```

Reply:
It treats includes as groups (a group is defined by new line). In each group it wants the includes to be alphabetically ordered and because it uses case sensitive it says that `R` should be before `c`. To prevent this I usually create a STL group before including anything from the library.

Reply:
Maybe it is a good idea to do this https://clang.llvm.org/docs/ClangFormatStyleOptions.html#sortincludes, what do you think?

Reply:
Isn't it for groups?

Reply:
It is.

Reply:
Case sensitive sorting is probably more convenient.